util/chplenv/chpl_comm_substrate.py

```diff
@@ -14,14 +14,18 @@ def get():
 
         if comm_val == 'gasnet':
             network = chpl_comm.get_network()
-            if network == 'ibv':
+            if network == 'aries':
+                substrate_val = 'aries'
```

Comment:
IIUC, this is defaulting the `substrate_val` to a value that no longer exists.

How about:
```suggestion
                substrate_val = 'ofi'
```

Reply:
This is intentionally defaulting to aries, so that later logic results in an error without action by a user (prompting them to use some other substrate, or preferably ugni). Given that ofi/mpi on XC is untested by GASNet and Chapel and that the XC is not officially supported (although Chapel does still support it), I am hesitant to add a new default.

Reply:
ok that makes sense, thanks.